auth/whoami.go

```diff
@@ -38,13 +37,16 @@ func printUserInfo(userInfo *api.UserInfo, cfg *Config) {
 
 	fmt.Printf("Your current organization: %q\n", cfg.Organization)
 
-	if len(userInfo.Orgs) > 1 {
+	// TODO: temporarily changed. Do we need to redesign this block?
```

Comment:
Fine with me to show this if there is only one organization available. We should just remove the comment.